java/src/jmri/jmrit/symbolicprog/DecVariableValue.java

```diff
@@ -19,18 +19,26 @@
 /**
  * Decimal representation of a value.
  *
- * @author Bob Jacobsen Copyright (C) 2001
+ * @author Bob Jacobsen Copyright (C) 2001, 2022
  */
 public class DecVariableValue extends VariableValue
         implements ActionListener, FocusListener {
 
-    public DecVariableValue(String name, String comment, String cvName,
-            boolean readOnly, boolean infoOnly, boolean writeOnly, boolean opsOnly,
-            String cvNum, String mask, int minVal, int maxVal,
-            HashMap<String, CvValue> v, JLabel status, String stdname) {
+    public DecVariableValue(String name, String comment, String cvName, boolean readOnly, boolean infoOnly,
+                            boolean writeOnly, boolean opsOnly, String cvNum, String mask, int minVal, int maxVal,
```

Comment:
See my https://github.com/JMRI/JMRI/pull/11281#issuecomment-1206050979

Reply:
My eyesight isn't up to checking the details but if you've reverted the addition of factor and offset to DecVariableValue I'm happy with that.